packages/blade/src/components/Dropdown/Dropdown.tsx

```diff
@@ -144,6 +149,7 @@ const _Dropdown = ({
       setIsKeydownPressed,
       dropdownBaseId,
       triggererRef,
+      triggererWrapperRef,
```

Comment:
this needs to be set by consumers?

Reply:
nope its internally set on Select and AutoComplete